nginx/servers_config.sls

```diff
@@ -138,7 +144,7 @@ nginx_server_available_dir:
 {% else %}
 {{ manage_status(server, settings.enabled, False) }}
 {% endif %}
-{% if settings.config != None and settings.enabled == True %}
+{% if settings.enabled == True %}
```

Comment:
I haven't looked into this but is there a reason you removed the `settings.config != None`?

Reply:
I removed this to be able to not specify a config object so that flat files can be retrieved by TOFS without this param set empty